reviewing/dependencies-hierarchy/test/getTree.test.ts

```diff
@@ -185,8 +185,7 @@ describe('getTree', () => {
 
       const result = getTree({
         ...commonMockGetTreeArgs,
-        currentDepth: 0,
-        maxDepth: 2,
+        maxDepth: 3,
```

Comment:
A separate commitaa5ecfaf5cc936d27c6e5acbf217236a8b745126 starts `currentDepth` at `1` instead of `0` in tests. Here's a copy of the commit message:

> Calls to `getTree` from the `buildDependenciesHierarchy` function always start the `currentDepth` at `1` rather than `0`. Updating `currentDepth` calls in test to also start at `1` for consistency with production code. This required incrementing the `maxDepth` argument in a few cases as well.